tracing/src/main/java/com/palantir/tracing/Tracers.java

```diff
@@ -26,6 +26,7 @@
 public final class Tracers {
     /** The key under which trace ids are inserted into SLF4J {@link org.slf4j.MDC MDCs}. */
     public static final String TRACE_ID_KEY = "traceId";
+    public static final String TRACE_SAMPLED_KEY = "traceSampled";
```

Comment:
We should document this key similarly to `TRACE_ID_KEY` and describe possible values.